docs/api/public_php_api_creating_content.md

```diff
@@ -1,377 +1,151 @@
+
+            $draft = $this->contentService->createContent($contentCreateStruct, [$locationCreateStruct]);
+
+            $output->writeln("Created a draft of " . $contentType->getName() . " with name " . $draft->getName());
+
+        } catch //..
+    }
 }
 ```
 
-## Creating Content containing an image
+This command creates a draft using [`ContentService::createContent`](https://github.com/ezsystems/ezpublish-kernel/blob/v7.5.3/eZ/Publish/API/Repository/ContentService.php#L204) (line 22).
+This method must receive a `ContentCreateStruct` and an array of Location structs.
 
-!!! note "Full code"
+`ContentCreateStruct` (which extends `ContentStruct`) is created through [`ContentService::newContentCreateStruct`](https://github.com/ezsystems/ezpublish-kernel/blob/v7.5.3/eZ/Publish/API/Repository/ContentService.php#L486) (line 18),
+which receives the Content Type and the primary language for the Content item.
+(See [Translating content](#translating-content) for information about translating the Content item into other languages).
 
-    <https://github.com/ezsystems/CookbookBundle/blob/master/Command/CreateImageCommand.php>
+[`ContentStruct::setField`](https://github.com/ezsystems/ezpublish-kernel/blob/v7.5.3/eZ/Publish/API/Repository/Values/Content/ContentStruct.php#L31) (line 19) enables you to define the Field values.
+When the Field accepts a simple value, you can provide it directly, as in the example above.
+For some Field Types, for example [images](#creating-an-image), you need to provide an instance of a Value type.
 
-As explained above, the `setField()` method can accept various values: an instance of the Field Type's Value class, a primitive type, or a hash. The last two depend on what the `Type::acceptValue()` method is build up to handle. TextLine can, for instance, accept a simple string as an input value. In this example, you will see how to set an Image value.
+### Creating an image
 
-Let's assume that you use the default image class. Creating your Content, using the Content Type and a `ContentCreateStruct`, has been covered above, and can be found in the full code. Let's focus on how the image is provided.
+Image Field Type requires providing an instance of its Value type to the [`ContentStruct::setField`](https://github.com/ezsystems/ezpublish-kernel/blob/v7.5.3/eZ/Publish/API/Repository/Values/Content/ContentStruct.php#L31) method.
+Therefore, to create a Content item of the Image type (or any other Content Type with an `image` Field Type),
+the `ContentCreateStruct` is slightly more complex then in the previous example:
 
 ``` php
-$file = '/path/to/image.png';
-
-$value = new \eZ\Publish\Core\FieldType\Image\Value(
+$contentType = $this->contentTypeService->loadContentTypeByIdentifier('image');
+$contentCreateStruct = $this->contentService->newContentCreateStruct($contentType, 'eng-GB');
+$contentCreateStruct->setField('name', $name);
```

Comment:
lets either use `My image` instead of `$name` (prefer imo), or declare `$name = 'My image'` under `$file`